test/NuGet.Core.Tests/NuGet.Packaging.Extraction.Test/NuGet.Packaging.Extraction.Test.csproj

```diff
@@ -0,0 +1,28 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.test.props" />
```

Comment:
Why do we need the extra project? Are the tests specific to NuGet.Packaging.Extraction? 

Can we run the tests in NuGet.Packaging.Test instead?

Reply:
You're right.  When I looked at in VS IDE, it showed NuGet.Packaging.Extraction as the "current project":
![image](https://user-images.githubusercontent.com/12734758/100678231-93b54580-3321-11eb-89e0-4aa76e92eee8.png)
...and I didn't realize that the NuGet.Packaging.Extraction project just links the files from NuGet.Packaging.  Your suggestion will simplify the PR.